Elmah.Io.Extensions.Logging/ElmahIoLoggerFactoryExtensions.cs

```diff
@@ -10,5 +10,11 @@ public static ILoggerFactory AddElmahIo(this ILoggerFactory factory, string apiK
             factory.AddProvider(new ElmahIoLoggerProvider(apiKey, logId));
             return factory;
         }
+
+        public static ILoggerFactory AddElmahIoWithFilter(this ILoggerFactory factory, string apiKey, Guid logId, FilterLoggerSettings filter)
```

Comment:
Would prefer if it was just an overload of `AddElmahIo`. This makes it consistent with the method naming in console etc.